### ISSUE TITLE
Storage Analysis

### ISSUE THREAD

**Issue by hez2010** [files-community/Files#6091]
### Solution/Idea

We can add a tool for storage analysis and drawing a graph helping users identify how files are taking up user's storage. 
Similar products: TreeMapFs. 

### Alternatives

N/A

### Files Version

v2.0

### Windows Version

Windows 11 22449.1000

**Comment by R3voA3** [comment]
Good idea. Here is a another app to get an idea https://www.microsoft.com/store/productId/9PFXCD722M2C

**Comment by yaira2** [comment]
Is this something that should be in the properties window?

Fyi @chingucoding

**Comment by yaira2** [comment]
Fyi @mdtauk
